dask_cuda/worker_spec.py

```diff
@@ -14,6 +14,7 @@ def worker_spec(
     threads_per_worker=1,
     silence_logs=True,
     CUDA_VISIBLE_DEVICES=None,
+    enable_tcp_over_ucx=False,
     enable_infiniband=False,
     ucx_net_devices="",
     enable_nvlink=False,
```

Comment:
Why wouldn't all of the `enable_foo` options be `True` by default?

Reply:
Only because I don't know if they're all safe in any combinations. I guess we can fine tune this to a "fast and safe" approach when we are confident everything should work.